src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -239,14 +239,26 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     /* hash */ resourceLoader.bootConfig.resources.runtime[dotnetWasmResourceName],
     /* type */ 'dotnetwasm');
 
-  const dotnetTimeZoneResourceName = 'dotnet.timezones.dat';
+  const dotnetTimeZoneResourceName = 'dotnet.timezones.blat';
```

Comment:
Is this a typo or a legit file format?

Reply:
I've heard Larry talk about this so I guess it's the name of the archive file format. Apparently it's like a `tar` file.